lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+  },
+  "not-found-title": {
+    "defaultMessage": "No s'ha trobat cap micro:bit",
+    "description": "Title for the micro:bit found dialog"
+  },
+  "not-found-update-link": {
+    "defaultMessage": "Has d'actualitzar el teu microprogramari <link>update your firmware</link> abans de poder connectar-te a aquesta micro:bit.",
+    "description": "Text in the no micro:bit found dialog"
+  },
+  "open-action": {
+    "defaultMessage": "Obert",
+    "description": "Open button text"
+  },
+  "open-file-action": {
+    "defaultMessage": "Obert",
+    "description": "Open file button text"
+  },
+  "open-file-dropped": {
+    "defaultMessage": "Obre el fitxer quan es deixa anar",
+    "description": "Aria label for file drop target"
+  },
+  "open-hover": {
+    "defaultMessage": "Obre un fitxer hexadecimal o Python o afegeix altres fitxers",
+    "description": "Hover text over load button"
+  },
+  "options": {
+    "defaultMessage": "Opcions",
+    "description": "Label for an options menu"
+  },
+  "parameter-help": {
+    "defaultMessage": "Ajuda dels paràmetres",
+    "description": "Setting label to control whether pop-up documentation for function/method parameters is automatically shown."
+  },
+  "parameter-help-automatic": {
+    "defaultMessage": "Automàtic",
+    "description": "Parameter help setting for when the parameter documentation is shown automatically"
+  },
+  "parameter-help-manual": {
+    "defaultMessage": "Manual ({shortcut})",
+    "description": "Parameter help setting for when the user must press a key combination to open the parameter documentation"
+  },
+  "permanently-delete": {
+    "defaultMessage": "Suprimir {filename} permanentment?",
+    "description": "Confirmation question to permanently delete file"
+  },
+  "post-save-message-files": {
+    "defaultMessage": "Conté tots els fitxers d'aquest projecte (inclosos els fitxers addicionals, per exemple, per executar accessoris, que s'enumeren a la pestanya <strong>Projecte</strong>).",
+    "description": "Message in dialog shown after the user saves the project as a hex file."
+  },
+  "post-save-message-one": {
+    "defaultMessage": "Trobaràs el teu fitxer hexadecimal a la carpeta <strong>Baixades</strong> d'aquest ordinador.",
+    "description": "Message in dialog shown after the user saves the project as a hex file."
+  },
+  "post-save-message-two": {
+    "defaultMessage": "Pots moure'l a una altra carpeta per emmagatzemar-lo i fer servir  <strong>Obre</strong> per continuar editant-lo més tard.",
+    "description": "Message in dialog shown after the user saves the project as a hex file."
+  },
+  "post-save-title": {
+    "defaultMessage": "Projecte desat",
+    "description": "Title of dialog shown after the user saves the project as a hex file."
+  },
+  "post-save-transfer-hex": {
+    "defaultMessage": "Per executar aquest fitxer hexadecimal a la teva micro:bit segueix aquests passos <link>follow these steps</link>.",
```

Comment:
Again, translate text inside link, don't duplicate.

Reply:
Requested in Crowdin